e2etest/clean/clean_distributor.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"log"
+	"strconv"
+)
+
+type documentVersionTuple struct {
+	str *string
+	num int
+}
+
+func main() {
+	// set up aws go sdk ssm client
+	testSession, err := session.NewSession()
+	if err != nil {
+		log.Fatalf("Error creating session %v", err)
+	}
+	ssmClient := ssm.New(testSession)
+
+	//get list of documents with versions
+	documentName := "testAWSDistroOTel-Collector"
+	listDocumentVersionInput := ssm.ListDocumentVersionsInput{Name: &documentName}
+	listDocumentVersionsOutput, err := ssmClient.ListDocumentVersions(&listDocumentVersionInput)
+	if err != nil {
+		log.Fatalf("Error getting document list %v", err)
+	} else if len(listDocumentVersionsOutput.DocumentVersions) < 1{
```

Comment:
```suggestion
	} else if len(listDocumentVersionsOutput.DocumentVersions) <= 1{
```
This doesn't need to fail, but it should exit early if there's only 1 or less, which will be the case often if this runs hourly. It shouldn't be a failure if we don't need to clean it up.